docs/developer-access-policy.md

```diff
@@ -0,0 +1,100 @@
+# Policy for OpenSAFELY Access by Platform Developers
+
+## Introduction
+
+OpenSAFELY securely mediates access to pseudonymized patient data for the purpose of analysis (be that research, service evaluation or audit).
+Our objective is to minimise direct access to pseudonymised data wherever possible, to protect privacy;
+while facilitating high quality, efficient analytics on the data by a wide range of users.
+A small number of OpenSAFELY team members need more direct access to the raw pseudonymised data inside the secure environment
+for the purposes of data curation and for developing and running the OpenSAFELY platform.
+
+This policy describes the conditions under which individuals with these roles may access the OpenSAFELY platform
+and the controls that are in place to ensure that such access cannot be used to subvert the OpenSAFELY security model.
+All such access conforms to the OpenSAFELY platform’s principles of security, transparency and minimal access.
+
+The specific types of access activities this policy describes are:
+
+* Running OpenSAFELY test pipelines
+* Ad hoc access to backend systems
+* Data curation activities
+
+## Running OpenSAFELY test pipelines
+
+### Why
+
+For software maintenance and service provision, it is sometimes necessary to execute OpenSAFELY [study pipelines](https://docs.opensafely.org/actions-pipelines/#project-pipelines).
+In particular this is necessary to ensure that the platform as a whole is operating correctly and to verify that changes to the software work correctly in the deployed system.
+
+### What
+
+The OpenSAFELY team maintains a small set of test studies in the [OpenSAFELY Testing](https://jobs.opensafely.org/datalab/opensafely-testing/) project.
+These studies have pipelines that use the standard OpenSAFELY tooling,
+but which have been specially designed to test and assess the platform’s functionality rather than to answer research questions.
+
+Because they are standard OpenSAFELY pipelines, they benefit from all the same security controls as OpenSAFELY research pipelines,
+including publication of all code that is run, logging of all operations and manual checking of outputs for disclosivity before they are released from the backend.
+Access to the OpenSAFELY platform will also be required to verify the pipeline outputs are as expected.
+
+### Who
+
+The test studies can only be run by OpenSAFELY Core Developers.
+
+### When
+
+These test pipelines are run periodically as is necessary to check that the system is working properly.
+They are also run when updated software has been deployed;
+however this deployment only happens after new versions of the software have gone through careful testing with simulated data to ensure that they are functioning correctly and that the security characteristics of the platform are maintained.
+
+## _Ad hoc_ access to backend systems
```

Comment:
Nit: Inconsistent italicization. IMHO we don't need to italicize it, because it's a word that's now entered the English language and is no longer on loan from Latin.